### PR TITLE
Number component creation

### LINKED ISSUE #31: Number component

### PR DESCRIPTION
**Pull request by andres-arista**
Closes #31

### CODE REVIEW

File: src/components/number/Number.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import imageAdd from '../../shared/images/icons/add.svg';
+import NumberProps from './Number.types';
+
+const Number: React.FC<NumberProps> = ({
+    value = 0,
+    label,
+    error,
+    className,
+    handleSubstract,
+    handleAdd,
```
Comment (BrandonDenham) [inline src/components/number/Number.tsx:23]: these shouldn't be add and subtract as props, add and subtract should call onChange

File: src/components/number/Number.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}) => {
+    const theme = useTheme();
```
Comment (BrandonDenham) [inline src/components/number/Number.tsx:25]: They should still be able to change the value of the number by entering the value in an input by typing . . . there should be an onChange that gets called.

**Review by BrandonDenham — CHANGES_REQUESTED** [review]

File: src/components/number/Number.tsx (hunk 1 above)
Comment (andres-arista) [reply src/components/number/Number.tsx]: Fixed

File: src/components/number/Number.tsx (hunk 2 above)
Comment (andres-arista) [reply src/components/number/Number.tsx:36]: Fixed

**Review by BrandonDenham — APPROVED** [review]

**Pull request closed without merging**